index.html

```diff
@@ -181,17 +182,24 @@ <h4 class="card__title">Skills</h4>
                 <h2 class="heading-2">I'm always interested in hearing about new projects, so if you'd like to chat, please get in touch.</h2>
             </div>
             <div class="contact__form">
-                <form>
+                <form action="https://formspree.io/f/mrgdrewl"
+                method="POST">
                     <div class="form-control">
-                        <input type="text" placeholder="Full name">
+                        <label>
+                            <input name="full_name" type="text" placeholder="Full name" required maxlength="30">
+                        </label>
                     </div>
                     <div class="form-control">
-                        <input type="email" placeholder="Email Address">
+                        <label>
+                            <input name="email" type="email" placeholder="Email Address" required>
+                        </label>
                     </div>
                     <div class="form-control">
-                        <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
+                        <label>
+                            <textarea name="message" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
+                        </label>
                     </div>
-                    <button class="btn">Get in touch</button>
+                    <button class="btn btn__submit" type="submit">Get in touch</button>
                 </form>
```

Comment:
- According to the project requirements, please use the `flex` display property to align the content of your contact in CSS, This allows for more flexibility and responsiveness 😄 😄